comprehensive-review/solutions/garden-streams/src/main/java/com/redhat/garden/sensors/Sensor.java

```diff
@@ -0,0 +1,15 @@
+package com.redhat.garden.sensors;
+
+public class Sensor {
```

Comment:
Is this an event? a model? I'm not 100% sure about the disposition of files in this sensors folder. If this is an event it should be located with the other events. We can have an package inside the events for each "type" if we want.